docs/_static/css/dark.css

```diff
@@ -626,12 +626,11 @@ code.docutils.literal.notranslate {
 }
 
 
-/* Literal.Number.Integer.Long */
-
-
 /* Link to Remix IDE shown over code snippets */
-a.remix-link {
-    filter: invert(1); /* The icon is black. In dark mode we want it white. */
+
+a.remix-link,
+a.remix-link:not(:visited) {
+    color: black !important;
 }
```

Comment:
I think this breaks the styling for visited Remix links. At least in the preview I can't see any difference in appearance between visited and non-visited ones.

Reply:
The styling for visited Remix links is specified by `a.remix-link`, so I think this doesn't break it. The reason why I added `a.remix-link:not(:visited)` is to overwrite the following style:

```css
.rst-content a:not(:visited) {
    color: #aaddff !important;
}
```

In other words, if this code doesn't exist, `a.remix-link:not(:visited)` is not necessary.

Reply:
There may be a better selector.

Reply:
For example:
```
.remix-link-container a.remix-link {
    color: black !important;
}
```

Reply:
> The styling for visited Remix links is specified by `a.remix-link`, so I think this doesn't break it.

It does, at least in Firefox. It overrides color on both `a` and `a:visited`. I think you need to style `a.remix-link:visited` separately.

By the way, forcing it with `!important` isn't really needed, is it? The rule still applies when I remove it.

Reply:
Ah, I guess you're using dark mode? Haven't really checked that. It's broken in the normal mode.

And I see that the rule you pointed out has `!important`, which I guess is why you needed it here. But (if possible) it would be better to just remove it from the original rule instead. The problem with `!important` is that it's quick and easy way initially, but once it's on a lot of rules, it's chaos :)

Reply:
Do you mean that clicked open-in-remix buttons should have a separate style? This button is represented by `<a>`, but I consider this a button, not a link (This may be contrary to HTML conventions, and maybe we should use `<button>`, but I don't know how to use it in Sphinx), so I think that colors don't need to be separated. Should we separate the colors?

>By the way, forcing it with !important isn't really needed, is it? The rule still applies when I remove it.

It's `a.remix-link, a.remix-link:not(:visited)` in dark mode, right? In my environment (Chrome), without the `!important`, the text color of the button is light blue. By the way, I don't quite like adding `!important`. I would remove them if I could. (There are `!importants` all over the CSS files before this change.)

![image](https://user-images.githubusercontent.com/20497787/232351249-5a761c50-b76c-4c61-b2a3-e0908d323a75.png)

Reply:
You may know, but just in case, if you changed the CSS files, `make clean` is required before `make html`.

Reply:
Sorry! GitHub didn't show your last reply. Please ignore the two comments above :)

Reply:
I thought you were talking about dark.css. How does it break in the normal mode? This works normally in my Chrome and Firefox ... Could you give me an image of it?

Reply:
I've deleted the `!important`s.

I also removed `!important` from the custom.css. It's a small change, so I included it in this PR. There are a lot of `!importants` in dark.css, so I didn't change it in this.